src/panels/lovelace/views/hui-view-editable.ts

```diff
@@ -1,3 +1,5 @@
 // hui-view dependencies for when in edit mode.
 import "../components/hui-card-options";
 import "../../../components/ha-fab";
+import Muuri from "muuri/dist/muuri";
```

Comment:
```suggestion
export Muuri from "muuri/dist/muuri";
```

Reply:
I don't think this is valid, is it? I receive many errors not just for the types not being included